src/app/tl-header/tl-header.component.html

```diff
@@ -279,6 +279,12 @@
 
 <ng-template #bottomNavContent>
     <ng-content select="wvre-nav-li"></ng-content>
+    <wvr-nav-li-component class="ml-auto supportTheLibraries">
+      <a href="about/giving/index.html">
```

Comment:
This should use a global url instead of relative.

Reply:
Global url used